R-package/README.md

```diff
@@ -265,10 +265,13 @@ The example below shows how to generate code coverage for the R package on a mac
 
 ```shell
 # Install
-sh build-cran-package.sh
+sh build-cran-package.sh \
+    --no-build-vignettes
```

Comment:
This change isn't necessary to make calculating test coverage work, but it does speed up the process a lot.

Since these instructions are only about computing the coverage from *tests*, it isn't necessary to build the vignettes.

https://github.com/microsoft/LightGBM/blob/af5b40e1f653e25a5f3470e09de4d1c68dc59663/R-package/README.md?plain=1#L272

I found that skipping vignette building saved on the order of 60-90 seconds in the process of computing coverage.